app/src/main/java/com/zeapo/pwdstore/utils/Extensions.kt

```diff
@@ -123,3 +124,7 @@ fun <T : View> AlertDialog.requestInputFocusOnView(@IdRes id: Int) {
 val Context.autofillManager: AutofillManager?
     @RequiresApi(Build.VERSION_CODES.O)
     get() = getSystemService()
+
+fun Activity.isInsideRepository(file: File): Boolean {
+    return file.canonicalPath.contains(getRepositoryDirectory(this).canonicalPath)
+}
```

Comment:
needs a trailing newline

Reply:
You mean I should put another line above the function?

Reply:
No, at the end of the file.